async-query/src/main/java/org/opensearch/sql/spark/cluster/ClusterManagerEventListener.java

```diff
@@ -183,7 +183,7 @@ private void cancel(Cancellable cron) {
 
   @VisibleForTesting
   public List<Cancellable> getFlintIndexRetentionCron() {
-    return Arrays.asList(flintIndexRetentionCron);
+    return Collections.singletonList(flintIndexRetentionCron);
```

Comment:
can u use static import for the `Collections.singletonList` method ?

Reply:
Added new commit that replaces this:

```
import java.util.Collections;
...
Collections.singletonList(...);
```

with static imports:
```
import static java.util.Collections.singletonList;
...
singletonList(...);
```

Reply:
See comment: https://github.com/opensearch-project/sql/pull/3193#discussion_r1876793204